### ISSUE TITLE
android:selectable not working

### ISSUE THREAD

**Issue by felipecds** [consp1racy/android-support-preference#84]
On previous versions of Support Library, the android:selectable attr for SwitchPreference worked flawlessly, but since version 27, the attr is not working. Only the enabled attr is working, but the switch are grayed (the default behavior).

Are there any way for removing the grayed style and show it like it's enable, but in a way that the user can't select or change the switch's state?

**Comment by consp1racy** [comment]
Hi, thanks for reporting!

This is my fault, caused by setting long click listener. It turns out even if you call `setLongClickListener(null)` it makes the view long clickable, and it looks and acts as clickable.

I'm preparing big changes in upcoming version 2.0.0 so I won't make a minor release with a fix just now.

You can work around this yourself in the mean time.

1. Subclass the preference you need.
2. Override its `onBindViewHolder(PreferenceViewHolder)` like so:

```java
@Override
public void onBindViewHolder(PreferenceViewHolder holder) {
    super.onBindViewHolder(holder);
    holder.itemView.setLongClickable(hasOnPreferenceLongClickListener() && isSelectable());
}
```

You can only do this for preferences in package `net.xpece.android.support.preference`.

3. Use fully qualified class name of your new class when referencing in XML.

**Comment by consp1racy** [comment]
Fixed in 540d1ab, released in v1.3.2.
